bcipy/signal/model/offline_analysis.py

```diff
@@ -527,17 +533,32 @@ def offline_analysis(
     for raw_data_path in data_file_paths:
         raw_data = load_raw_data(raw_data_path)
         device_spec = devices_by_name.get(raw_data.daq_type)
-        # extract relevant information from raw data object eeg
+            # extract relevant information from raw data object eeg
+        # if device_spec.content_type == "EEG" and device_spec.status == "active":
         if device_spec.content_type == "EEG":
+            eeg_data = raw_data
             erp_model = analyze_erp(
-                raw_data, parameters, device_spec, data_folder, estimate_balanced_acc, save_figures, show_figures)
+                raw_data,
+                parameters,
+                device_spec,
+                data_folder,
+                estimate_balanced_acc,
+                save_figures,
+                show_figures)
             models.append(erp_model)
 
+        # if device_spec.content_type == "Eyetracker" and device_spec.status == "active":
         if device_spec.content_type == "Eyetracker":
```

Comment:
We do need an is_active check here to prevent training the model if it's passive

Reply:
I was explaining this earlier that it was a workaround for an error I got, because `device_spec.status` property does not exist in the old datasets. I talked to Matt now I am using `device_spec.is_active` property to check their status.